bigbang/mailman.py

```diff
@@ -451,7 +453,7 @@ def messages_to_dataframe(messages):
     indexed by message-id, with column-names from headers.
     """
     def safe_unicode(t):
-        return t and unicode(t, 'utf-8', 'replace')
+        return t
```

Comment:
This method is a no-op now, so presumably we should re-factor it away.

But also! Email headers are their own classes now, so we actually need to call `str` in all the places we were previously calling `safe_unicode` to get the final string representation of these headers, otherwise we get errors about how a Header object doesn't have a replace method, or that a Header object isn't hashable.

So either: remove `safe_unicode` method and replace each call with `str()` or have `safe_unicode` `return str(t)`.